src/api/searchApi.ts

```diff
@@ -414,7 +414,7 @@ export const fetchResults = async (params: FetchResultsParams, signal?: AbortSig
 
 export const fetchRegistrationsExport = async (searchParams: URLSearchParams) => {
   searchParams.set(ResultParam.From, '0');
-  searchParams.set(ResultParam.Results, '1000');
+  searchParams.set(ResultParam.Results, '400');
```

Comment:
Backend crashes at around 500-600 depending on size of exported items.
I will look into improving the backend architecture to support unlimited, but in the meantime it would be nice if this export does not crash